src/hooks/useFetch.ts

```diff
@@ -21,7 +21,12 @@ export interface State<T> {
 
 export interface FetchResponse<T> extends State<T> {
   del: (url: string, requestOptions?: Data) => Promise<void>
-  get: (url: string, params?: Data, requestOptions?: Data) => Promise<void>
+  get: (
+    url: string,
+    params?: Data,
```

Comment:
This is not code that I've touched. I just added the optionalHaeders in a similar fashion. If I was to change it, the arguments need to be changed around all over the code since a mandatory argument cannot be followed by an optional.

Reply:
aaah okay